cluster-proxy/src/main/java/com/sequenceiq/cloudbreak/clusterproxy/ConfigRegistrationRequestBuilder.java

```diff
@@ -0,0 +1,52 @@
+package com.sequenceiq.cloudbreak.clusterproxy;
+
+import java.util.List;
+
+public class ConfigRegistrationRequestBuilder {
+    private String accountId;
+
+    private String clusterCrn;
+
+    private String uriOfKnox;
+
+    private List<String> aliases;
+
+    private List<ClusterServiceConfig> services;
+
+    private List<String> certificates;
+
+    private boolean useTunnel;
+
+    private List<TunnelEntry> tunnels;
+
+    public ConfigRegistrationRequestBuilder(String clusterCrn) {
+        this.clusterCrn = clusterCrn;
+    }
+
+    public ConfigRegistrationRequestBuilder with(List<String> aliases, List<ClusterServiceConfig> services, List<String> certificates) {
```

Comment:
this is a little bit strange for me to have a `with` with multiple params, usually builders have separate method for every of them

Reply:
these are the minimal required parameters so thought this should be ok

Reply:
then I think it should be moved to the constructor or the build method must have some check